lib/Extension/ClassMover/Application/Finder/FileFinder.php

```diff
@@ -78,35 +80,68 @@ private function pathsFromReflectionClass(ReflectionClass $reflection, bool $pri
         return FileList::fromFilePaths($filePaths);
     }
 
-    private function allPhpFiles(Filesystem $filesystem)
+    private function allPhpFiles(Filesystem $filesystem): FileList
     {
-        $filePaths = $filesystem->fileList()->existing()->phpFiles();
-        return $filePaths;
+        return $filesystem->fileList()->existing()->phpFiles();
     }
 
-    private function parentFilePaths(ReflectionClass $reflection, $filePaths)
+    /**
+     * @param array<string> $filePaths
+     *
+     * @return array<string>
```

Comment:
this was `array<string|null>`? is this definitely not handled? we could also skip if it's NULL for whatever reason.

Reply:
Yeah seeing that this never crashes it seems like it's never null but yeah, skipping those would probably be the better option.